typings/blockly.d.ts

```diff
@@ -1036,6 +1036,63 @@ declare module "serialization/exceptions" {
     }
     import { Block } from "block";
 }
+
+declare module "serialization/variables" {
+    /**
+     * Represents the state of a given variable.
+     */
+    export type State = {
+        name: string;
+        id: string;
+        type: (string | undefined);
+    };
+    /**
+     * Represents the state of a given variable.
+     * @typedef {{
+     *   name: string,
+     *   id: string,
+     *   type: (string|undefined)
+     * }}
+     * @alias Blockly.serialization.variables.State
+     */
+    export let State: any;
```

Comment:
I don't think this let is actually necessary, but probably best to leave as is. Iirc it was just to make lint & closure happy, but seems to have ended up here.

Reply:
I copy / pasted the block without editing. I agree it may not be necessary but wanted to keep my scope of changes limited.